app/nation/app/views/SurveyTable.js

```diff
@@ -17,7 +17,7 @@ $(function() {
         },
 
         addAll: function() {
-            this.$el.append('<tr><th>'+App.languageDictValue.get('Date')+'</th><th>'+App.languageDictValue.get("Survey_Number")+'</th><th>'+App.languageDictValue.get('Title')+'</th><th>'+App.languageDictValue.get('Sent_to')+'</th><th>'+App.languageDictValue.get('Submitted_By')+'</th><th>'+App.languageDictValue.get('Response')+'</th><th colspan="2">'+App.languageDictValue.get('Actions')+'</th></tr>')
+            this.$el.append('<tr><th>'+App.languageDictValue.get('Date')+'</th><th>'+App.languageDictValue.get("Survey_Number")+'</th><th>'+App.languageDictValue.get('Title')+'</th><th>'+App.languageDictValue.get('Sent_to')+'</th><th>'+App.languageDictValue.get('Submitted_By')+'</th><th>'+App.languageDictValue.get('Received')+'/'+App.languageDictValue.get('Sent')+'</th><th colspan="2">'+App.languageDictValue.get('Actions')+'</th></tr>')
```

Comment:
Please make sure that newly used words are added in int_docs folder as well; also that whether that word is already there or not. follow https://jsonformatter.curiousconcept.com/ to verify all 8 languages files.